rbi/gems/test_case.rbi

```diff
@@ -0,0 +1,12 @@
+# typed: __STDLIB_INTERNAL
+
+class ActiveSupport::TestCase < ::Minitest::Test
```

Comment:
`ActiveSupport` _definitely_ does not belong in Sorbet. we can duplicate this into any tests that need it.